helpers/index.ts

```diff
@@ -192,7 +193,7 @@ export const installTemplate = async (
     if (
       props.template === "streaming" ||
       props.template === "multiagent" ||
-      props.template === "extractor"
+      props.template === "reflex"
```

Comment:
Changed to `reflex` for sharing reflex code, the specified code for each use case moved to `component/agents` folder; as reflex is only for python so it's a bit tricky, also welcome for better ideas.

Reply:
how about just storing those components in `components/reflex`? as we might do something non-agent?

Reply:
@leehuwuj great idea!